readme.md

```diff
@@ -1,34 +1,42 @@
-# → Visit [uses.tech](https://uses.tech) for a good time
-
-A list of `/uses` pages detailing developer setups, gear, software and configs.
-
-Add your own `/uses` page in [data.js](https://github.com/wesbos/awesome-uses/blob/master/src/data.js).
-
-This readme is auto-generated from the data.js file, so please don't PR this file.
-
-```
-   ▄████████  ▄█     █▄     ▄████████    ▄████████  ▄██████▄    ▄▄▄▄███▄▄▄▄      ▄████████
-  ███    ███ ███     ███   ███    ███   ███    ███ ███    ███ ▄██▀▀▀███▀▀▀██▄   ███    ███
-  ███    ███ ███     ███   ███    █▀    ███    █▀  ███    ███ ███   ███   ███   ███    █▀
-  ███    ███ ███     ███  ▄███▄▄▄       ███        ███    ███ ███   ███   ███  ▄███▄▄▄
-▀███████████ ███     ███ ▀▀███▀▀▀     ▀███████████ ███    ███ ███   ███   ███ ▀▀███▀▀▀
-  ███    ███ ███     ███   ███    █▄           ███ ███    ███ ███   ███   ███   ███    █▄
-  ███    ███ ███ ▄█▄ ███   ███    ███    ▄█    ███ ███    ███ ███   ███   ███   ███    ███
-  ███    █▀   ▀███▀███▀    ██████████  ▄████████▀   ▀██████▀   ▀█   ███   █▀    ██████████
-
-███    █▄     ▄████████    ▄████████    ▄████████
-███    ███   ███    ███   ███    ███   ███    ███
-███    ███   ███    █▀    ███    █▀    ███    █▀
-███    ███   ███         ▄███▄▄▄       ███
-███    ███ ▀███████████ ▀▀███▀▀▀     ▀███████████
-███    ███          ███   ███    █▄           ███
-███    ███    ▄█    ███   ███    ███    ▄█    ███
-████████▀   ▄████████▀    ██████████  ▄████████▀
-
-```
-
-# Awesome Uses ![Awesome][awesome-badge]
-
+# → Visit [uses.tech](https://uses.tech) for a good time
+
+[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/wesbos/awesome-uses/blob/master/src/data.js)

```

Comment:
regardless of whether or not we want to integrate GitPod, this is the wrong file (it's auto-generated on pushes to master).

You want to edit ./scripts/readme-template.md instead